### PR TITLE
Adds a few updates

### PR DESCRIPTION
**Pull request by jvanderaa**
- Adds a few items that may fringe on automation/network.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,6 +335,14 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 - [OSPF Watcher](https://github.com/Vadims06/ospfwatcher) - Tracks OSPF topology changes--by establishing a GRE tunnel with network devices--via a history diagram.
 - [Topolograph](https://github.com/Vadims06/topolograph) - Python-based Web tool for visualisation of OSPF/ISIS topologies and making a prediction of network behaviour in case of network's outage.
 
+## Network Management Projects
+
+- [InfluxDB](https://www.influxdata.com/) - Time series database loosely based on SQL
+- [Prometheus](https://prometheus.io/) - Time series database with PromQL
+- [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) - Collector for time series metrics
+- [TimescaleDB](https://www.timescale.com/) - Time series database based on PostgreSQL
+-
```
Comment (itdependsnetworks) [inline README.md:344]: remove

File: README.md [hunk 2]
```diff
@@ -334,6 +335,14 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 - [OSPF Watcher](https://github.com/Vadims06/ospfwatcher) - Tracks OSPF topology changes--by establishing a GRE tunnel with network devices--via a history diagram.
 - [Topolograph](https://github.com/Vadims06/topolograph) - Python-based Web tool for visualisation of OSPF/ISIS topologies and making a prediction of network behaviour in case of network's outage.
 
+## Network Management Projects
```
Comment (itdependsnetworks) [inline README.md:338]: Network Telemetry? 

Can you add to the index?

File: README.md [hunk 3]
```diff
@@ -334,6 +335,14 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
+
+- [InfluxDB](https://www.influxdata.com/) - Time series database loosely based on SQL
```
Comment (itdependsnetworks) [inline README.md:340]: Can you add a period to end of the each?

Let'e make sure to make full sentences. Often I grab from the origin website.

File: README.md (hunk 1 above)
Comment (itdependsnetworks) [reply README.md:344]: remove.... please?

Comment (jvanderaa) [reply README.md:344]: The extra bullet remove, I thought it was the entire section...

Comment (jvanderaa) [reply README.md:344]: Which then made the other comments completely off.

**Pull request merged**